src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -249,12 +253,14 @@ export default {
 			if (this.scrollPos < 0) {
 				const newLeftMargin = Math.min(0, this.scrollPos + this.shiftIncrement);
 				this.scrollPos = newLeftMargin;
+				this.$emit('scrolling-row');
```

Comment:
This is pretty cool, it emits an event when we're scrolling the row. This was functional since we needed to disable the hover animation while the row is scrolling.